apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.component.ts

```diff
@@ -23,7 +33,8 @@ import { CreateOrUpdateAccountDialogParams } from './interfaces/interfaces';
 export class CreateOrUpdateAccountDialog implements OnDestroy {
   public accountForm: FormGroup;
   public currencies: string[] = [];
-  public platforms: { id: string; name: string }[];
+  public platforms: Platform[];
+  public filteredPlatforms: Observable<Platform[]>;
```

Comment:
Can you move up ⬆️ to keep the alphabetical order?

Reply:
done bd2b36622d4ccea0b9086ba6c7ba4f66be2010b3